messages/auth.md

```diff
@@ -0,0 +1,37 @@
+# defaultOrgNotFound
+
+No %s org found
+
+# defaultOrgNotFoundActions
+
+- Run the "sfdx auth" commands with --setdefaultusername to connect to an org and set it as your default org.
+- Run "force:org:create" with --setdefaultusername to create a scratch org and set it as your default org.
+- Run "sfdx config:set defaultusername=<username>" to set your default username.
+
```

Comment:
Question: will this message be displayed with actual bullets?  Or will the message contain the literal character "-" at the beginning of each line?  Just curious. We talked about some Markdown elements *not* rendering in the terminal output, it would be good for me to list either they stuff that does (or doesn't, depending on how long the list is) render using Markdown format in the terminal.

Reply:
This would just make an array of messages, for whatever the consumer wants to do with it. So the existing errors with actions would still look exactly the same. 

Since it is an array, a consumer can do whatever they want with it, meaning vscode could display it in whatever way that makes sense for them, etc.